test/util.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+
+DATATYPES_EDS = os.path.join(os.path.dirname(__file__), "datatypes.eds")
+SAMPLE_EDS = os.path.join(os.path.dirname(__file__), "sample.eds")
+EDS_PATH = SAMPLE_EDS
```

Comment:
Shouldn't we get rid of this old name everywhere?  Might just as well adjust the other usages to the new name.

Reply:
Sure; I just wanted to keep the diff to a minimum. I'll update the PR to consistently use the new name only.

Reply:
Addressed with 0905d017c69ee9f3727dc3480591ddb073fc746b.